backend/cfg/sub_cfg.mli

```diff
@@ -22,6 +22,18 @@
 
 [@@@ocaml.warning "+a-40-41-42"]
 
+(** These IDs are also used by [make_instr] *)
+val instr_id : InstructionId.sequence
+
+val next_instr_id : unit -> InstructionId.t
+
+val reset_instr_id : unit -> unit
```

Comment:
are these two functions unused? should `reset_instr_id` be called from `Cfg_selectgen.fundecl`?

Reply:
`next_instr_id` is used, but maybe should not be exported.
It looks like `reset_instr_id` was not used before this PR,
and should indeed maybe be called from
`Cfg_selectgen.fundecl`.

Reply:
> `next_instr_id` is used, but maybe should not be exported.

yes, that's what I meant.

>  It looks like `reset_instr_id` was not used before this PR, and should indeed maybe be called from `Cfg_selectgen.fundecl`.

either removing it or using it is fine.